loopchain/utils/__init__.py

```diff
@@ -391,23 +391,56 @@ def parse_target_list(targets: str) -> list:
     return target_list
 
 
-def init_default_key_value_store(store_identity) -> Tuple[KeyValueStore, str]:
+def rename_db_dir(new_dirname: str):
+    """Rename db directory
+    This method is temporary implementation for remove 'ip:port' in db directory name
+    Ths method is useless after rename all nodes done.
+
+    :param new_dirname:
+    :return:
+    """
+    storage_path = Path(conf.DEFAULT_STORAGE_PATH)
+    if os.path.exists(storage_path / new_dirname):
+        return
+
+    regex = re.compile(r'([0-9]{1,3}\.){3}[0-9]{1,3}:[0-9]{4}')
```

Comment:
I don't know why you fix the number of port digits to 4.
Some node can use non-default port which is less than 1000 or more than 9999.

Reply:
regex removed.